framework/src/modules/validators/schemas.ts

```diff
@@ -42,6 +46,8 @@ export const generatorListSchema = {
 };
 
 export const validatorAddressSchema = {
+	$id: '/validators/registeredBlsKeysSubStore',
+	title: 'Registered Validators Bls Keys',
```

Comment:
this looks off, I think its ?
```suggestion
	$id: '/validators/address',
	title: 'Validators Address',
```

Reply:
yes, I've changed the `title` to `Validators Addresses` but kept the `$id` same to follow the substore name defined in the LIP.